scripts/generate_teams.py

```diff
@@ -0,0 +1,183 @@
+# Description: This script generates the HTML for PyBaMM's maintainers and contributors
+# using the GitHub API.
+
+# The HTML is then used in the website's "Teams" page at https://pybamm.org/teams.
+# Mostly adapted from the scientific-python-hugo-theme teams generation code.
+
+import requests
+import string
+
+# Current maintainers and maintainer trainees, hard-coded in alphabetical order
+PYBAMM_MAINTAINERS = [
+    "brosaplanella",
+    "martinjrobins",
+    "priyanshuone6",
+    "rtimms",
+    "Saransh-cpp",
+    "Scottmar93",
+    "tinosulzer",
+    "TomTranter",
+]
```

Comment:
add emeritus maintainers (scott marquies, thibaut lestang)

Reply:
Scott Marquis is already there as Scottmar93, will add Thibault Lestang